benchmark/conftest.py

```diff
@@ -233,6 +232,9 @@ def conversation(request, model_name):
             prompts={},
             correct=False,
         )
+        # delete first dots if venv is in project env
```

Comment:
As mentioned, I would not do this. Rather use `.venv` and configure your setup to use this env reliably. Env setup is on the user, should not be mandated by the code.